tests/utils/exec/rosacli/machinepool_service.go

```diff
@@ -106,7 +106,7 @@ type NodePoolDescription struct {
 	ID                         string              `yaml:"ID,omitempty"`
 	ClusterID                  string              `yaml:"Cluster ID,omitempty"`
 	AutoScaling                string              `yaml:"Autoscaling,omitempty"`
-	DesiredReplicas            string              `yaml:"Desired replicas,omitempty"`
+	DesiredReplicas            interface{}         `yaml:"Desired replicas,omitempty"`
```

Comment:
You need to convert `replicasCount` to a string but not replace the type here. At least, it could a string or int but not interface{}